### DISCUSSION TITLE
Can not send Quarkus logs to remote ELK server

### DISCUSSION THREAD

**Discussion by nickhais** [quarkusio/quarkus#40212] [Q&A]
Hello team!

We are configuring Quarkus logs to be send to remote ELK server. For that we use this [doc](https://quarkus.io/guides/centralized-log-management).

So we've added to application.properties file
```
quarkus.log.handler.gelf.enabled=true
quarkus.log.handler.gelf.host=ELK_IP
quarkus.log.handler.gelf.port=ELK_HOST
quarkus.log.handler.gelf.include-full-mdc=true
```

and required dep:

```xml
<dependency>
    <groupId>io.quarkus</groupId>
    <artifactId>quarkus-logging-gelf</artifactId>
</dependency>
```

Meanwhile the following ENV vars were also added:

```
QUARKUS_LOG_HANDLER_GELF_ENABLED='true'
QUARKUS_LOG_HANDLER_GELF_HOST='ELK_IP'
QUARKUS_LOG_HANDLER_GELF_PORT='ELK_HOST'
```

and enabled UDP connection to ELK and added pipeline to support GELF format. Our applications runs in AWS EKS cluster and all infra changes were applied.

So after all code changes we build container locally and that it was deployed to our k8s cluster. I've checked and required env vars present for pod.

But still no logs are visible in ELK stack.

What have we missed?

P.S. doc says to use Java 17 but we use Java 11. Can it be an issue?

Best regards,
Nick.

**Comment by aakashk2910** [comment]
I'm facing the same issue. I can send logs via command line using nc command. However with Quarkus nothing is sent.
